sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.aest.controller.assessOfferingChangeRequest.e2e-spec.ts

```diff
@@ -87,7 +87,7 @@ describe("EducationProgramOfferingAESTController(e2e)-assessOfferingChangeReques
       .expect({
         statusCode: HttpStatus.UNPROCESSABLE_ENTITY,
         message:
-          "Either offering not found or the offering not in appropriate status to be approved or declined for change.",
+          "Either offering not found, program inactive or the offering not in appropriate status to be approved or declined for change.",
```

Comment:
Either offering not found  **or the offering or program** not in appropriate status to be approved or declined for change. ?